contrib/fwupd.spec.in

```diff
@@ -100,7 +100,6 @@ BuildRequires: libmbim-devel
 %endif
 
 %if 0%{?have_uefi}
-BuildRequires: efivar-devel >= 33
```

Comment:
something similar needs to be changed from `contrib/debian/rules`.

Reply:
I think I've done that.